<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Palette</class>
 <widget class="QFrame" name="Palette">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>71</width>
    <height>316</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QToolButton" name="tbbg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>15</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Background color</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#000000</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="tbfg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Foreground color</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#B2B2B2</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>50</y>
     <width>61</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="tbcol0">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#000000</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol8">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#555555</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="tbcol1">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#AA0000</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol9">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#FF5555</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QToolButton" name="tbcol2">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#00AA00</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol10">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#55FF55</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QToolButton" name="tbcol3">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#AA5500</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol11">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#FFFF55</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QToolButton" name="tbcol4">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#0000AA</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol12">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#5555FF</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QToolButton" name="tbcol5">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#AA00AA</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol13">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#FF55FF</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QToolButton" name="tbcol6">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#00AAAA</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol14">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#55FFFF</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="tbcol7">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#B2B2B2</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbcol15">
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#FFFFFF</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tbfg</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>tbfgClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbbg</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>tbbgClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol0</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col0Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol1</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col1Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol2</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col2Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol3</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col3Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol4</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col4Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol5</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col5Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol6</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col6Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol7</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col7Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol8</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col8Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol9</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col9Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol10</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col10Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol11</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col11Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol12</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col12Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol13</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col13Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol14</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col14Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbcol15</sender>
   <signal>released()</signal>
   <receiver>Palette</receiver>
   <slot>col15Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tbfgClick()</slot>
  <slot>tbbgClick()</slot>
  <slot>col0Click()</slot>
  <slot>col1Click()</slot>
  <slot>col2Click()</slot>
  <slot>col3Click()</slot>
  <slot>col4Click()</slot>
  <slot>col5Click()</slot>
  <slot>col6Click()</slot>
  <slot>col7Click()</slot>
  <slot>col8Click()</slot>
  <slot>col9Click()</slot>
  <slot>col10Click()</slot>
  <slot>col11Click()</slot>
  <slot>col12Click()</slot>
  <slot>col13Click()</slot>
  <slot>col14Click()</slot>
  <slot>col15Click()</slot>
 </slots>
</ui>
